### ISSUE TITLE
Remove "bot owner" command

### ISSUE THREAD

**Issue by kjsmita6** [SniperNoob95/NASABot#7]
/togglelogging is documented as for the owner of the bot to enable logging. This shouldn't be visible to anyone except that bot owner. If you can't leave it as a slash command, make it an old style command so people don't see it.

**Comment by SniperNoob95** [comment]
I don't think this is an issue, as only the owner can use it. I cannot use a non-slash command as verified bots may only use slash commands by design. It's not ideal to have it public, but there's no risk of exploitation. Even if someone were to find a way to use it, it would be harmless.
